aaf-rubocop.yml

```diff
@@ -4,13 +4,15 @@
 # you must add the pre-existing exceptions found here as well.
 
 AllCops:
-  TargetRubyVersion: 2.3
   Exclude:
     - db/schema.rb
 
 Rails:
   Enabled: true
 
+Rails/HasManyOrHasOneDependent:
+  Enabled: false
```

Comment:
Is this cop bogus?

Reply:
![screen shot 2017-09-29 at 2 08 34 pm](https://user-images.githubusercontent.com/144163/31000555-ae51e5ee-a51f-11e7-9f85-5cb603eb1a67.png)

I don't understand why you would need to declare `dependent: destroy` on every association.

Reply:
Forces you to specify what happens, rather than just relying on the default. It's not something that I'd really think to set as a hard rule, but I'm also not sure it's bogus enough to disable globally.

¯\\\_(ツ)\_/¯